### DISCUSSION TITLE
Connection issues AAPS - Omnipod Dash

### DISCUSSION THREAD

**Discussion by MaaikeLJ** [NightscoutFoundation/xDrip#2433] [Q&A]
Hi all,

I just started my journey on AAPS two months ago. Since I switched from FSL2 + Diabox to Dexcom One + xdrip, my AAPS is having serious issues connecting with my Omnipod Dash. Could someone try and help me to find a solution please?

I have: 
Dexcom One
Xdrip+ version 2022.09.21
AAPS 
Omnipod Dash
Nightscout
on a Xiaomi Redmi Note 8 pro (Android 10)

Before, I had FSL2 with Diabox, AAPS and Omnipod Dash on the same phone.

Situation: Xdrip is running. It gets the readings from the Dexcom and sends the data to Nightscout and to AAPS. AAPS is also working.

I have some errors in Xdrip concerning OB1 G5, but it's working (screenshot).

But since I set up the Dexcom with Xdrip, AAPS keeps losing connection with my Omnipod Dash (which was never a problem before with the FSL2/Diabox). It's very strange. Every +/-4 hours it loses the connection with my pump and I have to reboot the phone. Afterwards, everything works again.

But this means I get an alarm at least every 4 hours, also during the night. If I ignore it, AAPS can't give me SMB's anymore and can't change my TBR's.

All my settings are according to the instructions.

I have already tried:
- install other version of Xdrip (nightly instead of latest stable)
- change OB1 G5 settings
- run 'Don't kill my apps' (score of 100%)
- check all app permissions
- disable battery optimalisation 
- change the Omnipod pod
- disable Google Play Protect 

I'm a complete IT-noob, so I have absolutely no clue what might cause the problem. At this point, I'm getting a bit desperate. Could someone help me, please?
![inbound3151815578854193792](https://user-images.githubusercontent.com/117526474/200125311-68853347-5ce5-4bd5-bb5c-3600f99bb23e.jpg)

**Comment by jamorham** [comment]
The main issue I have seen with this brand of phone is the power saving shutting apps down at night. You mention don't kill my app but just to confirm that you've followed whatever options are appropriate as described here:
https://dontkillmyapp.com/xiaomi

Do you have anything else using bluetooth or other devices connecting to your CGM or Pump?

**Reply by MaaikeLJ** [reply]
Thank you for your answer.

I have tried everything there is written in the link you mention (don't kill my app - not everything was possible).

I have no othere devices connecting but the Dexcom and my pump
